test/functional/wallet_assumeutxo.py

```diff
@@ -52,6 +54,11 @@ def run_test(self):
         Load the snapshot into the second, ensure it syncs to tip and completes
         background validation when connected to the first.
         """
+        # Disable this test for windows currently because it doesn't allow two
+        # bitcoind processes simultaneously trying to read the same wallet file.
+        if platform.system() == "Windows":
+            return
```

Comment:
I think it would be better if we used `skip_if_no_module` so that the test is reported as skipped instead of silently "passing" on windows. Could also use `skip_if_platform_not_posix` for that.